crates/api_common/src/post.rs

```diff
@@ -30,6 +31,7 @@ pub struct CreatePost {
   pub language_id: Option<LanguageId>,
   /// Instead of fetching a thumbnail, use a custom one.
   pub custom_thumbnail: Option<String>,
+  pub scheduled_time: Option<DateTime<Utc>>,
```

Comment:
Use `Option<i64>` , as unix time is how we handle dates API request dates.

Reply:
Done